.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,181 @@
+name: Benchmark Xdebug performance
+
+on:
+    schedule:
```

Comment:
This workflow runs on three triggers:

On schedule, weekly
Manually
On pull requests

Unfortunately the schedule and manual trigger only work on the main branch, so I have been unable to test them. We should be able to test them once this PR is merged and can fix if there is any issue